ui/assets/jsonschema/schema-04.json

```diff
@@ -90,7 +90,7 @@
         "required": { "$ref": "#/definitions/stringArray" },
         "additionalProperties": {
             "anyOf": [
-                { "type": "boolean" },
+                { "type": "string" },
```

Comment:
why this??

Reply:
Related to: I have a query regarding this Single text field for [UI] Design Configurator: AdditionalProperties as single text field #4938

Reply:
@Chadha93 , this is a standard schema and we should not edit this. This is not directly related to what we see on the UI. Will you please revert this change 😅  ?

Reply:
Sure will revert the change.

Reply:
Fixed